common/app/views/fragments/meta/metaInline.scala.html

```diff
@@ -1,49 +1,37 @@
 @(item: model.ContentType, isAmp: Boolean = false)(implicit request: RequestHeader)
 
+
 @import common.{LinkTo, Localisation}
 @import conf.switches.Switches._
+@import model.Badges.badgeFor
 
-@defining(
-    USElectionSwitch.isSwitchedOn &&
-    item.content.tags.tags.exists(_.id == "us-news/us-elections-2016"),
-    AUSElectionSwitch.isSwitchedOn &&
-    item.content.tags.tags.exists(_.id == "australia-news/australian-election-2016")
-){ case (isUSElection, isAUSElection) =>
-    <div class="content__labels @if(isUSElection) {content__labels--us-election} @if(item.content.showNewGalleryDesign) {content__labels--gallery}">
-        @if(isUSElection) {
-            <div class="badge-slot badge-slot--us-election">
-                @if(isAmp) {
-                    <amp-img class="badge-slot__img" src="@Configuration.static.path/sys-images/Guardian/Pix/pictures/2016/2/2/1454424596176/USElectionlogooffset.png" layout="fixed" height="33" width="33"></amp-img>
-                } else {
-                    <img class="badge-slot__img" src="@Configuration.static.path/sys-images/Guardian/Pix/pictures/2016/2/2/1454424596176/USElectionlogooffset.png"/>
-                }
-            </div>
-        }
+<div class="content__labels @if(item.content.tags.isUSElection) {content__labels--us-election} @if(item.content.tags.isAusElection) {content__labels--aus-election} @if(item.content.showNewGalleryDesign) {content__labels--gallery}">
+    @if(USElectionSwitch.isSwitchedOn || AUSElectionSwitch.isSwitchedOn) {
 
-        @if(isAUSElection) {
-            <div class="badge-slot badge-slot--us-election">
+        @badgeFor(item).map { badge =>
+            <div class="badge-slot badge-slot--election">
                 @if(isAmp) {
-                    <amp-img class="badge-slot__img" src="@Static("images/AUSElectionBadge.png")" layout="fixed" height="33" width="33"></amp-img>
+                    <amp-img class="badge-slot__img" src="@badge.imageUrl" layout="fixed" height="33" width="33"></amp-img>
```

Comment:
Yeah, the stuff I did here is so we can add new badges without having to change templates.